<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="bgWidget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#bgWidget{
background-color: rgb(0, 0, 0);
}</string>
    </property>
    <widget class="QPushButton" name="backBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Roboto&quot;;</string>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mongoUrlInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Roboto&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mongoUrlTitle">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>MongoDB URL:</string>
     </property>
    </widget>
    <widget class="QLabel" name="databaseNameTitle">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>270</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Database Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="databaseNameInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>310</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Roboto&quot;;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="configDatabaseConnectBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Roboto&quot;;</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="dbConnectionFailedTitle">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Roboto&quot;;
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
